server/Dockerfile

```diff
@@ -78,40 +81,37 @@ FROM dev AS prod
 COPY server .
 RUN npm run build
 RUN npm prune --omit=dev --omit=optional
-COPY --from=dev /usr/src/app/node_modules/@img ./node_modules/@img
-COPY --from=dev /usr/src/app/node_modules/exiftool-vendored.pl ./node_modules/exiftool-vendored.pl
+COPY --from=dev /usr/src/app/server/node_modules/@img ./node_modules/@img
+COPY --from=dev /usr/src/app/server/node_modules/exiftool-vendored.pl ./node_modules/exiftool-vendored.pl
 
 # web build
 FROM node:22.16.0-alpine3.20@sha256:2289fb1fba0f4633b08ec47b94a89c7e20b829fc5679f9b7b298eaa2f1ed8b7e AS web
 
-WORKDIR /usr/src/open-api/typescript-sdk
-COPY open-api/typescript-sdk/package*.json open-api/typescript-sdk/tsconfig*.json ./
-RUN npm ci
-COPY open-api/typescript-sdk/ ./
-RUN npm run build
-
 WORKDIR /usr/src/app
-COPY web/package*.json web/svelte.config.js ./
-RUN npm ci
-COPY web ./
-COPY i18n ../i18n
-RUN npm run build
+COPY ./web ./web/
```

Comment:
I couldn't resist - doing a copy/run/copy/run creates a lot of unnecessary layers. Do all the copies, then do the runs. The two runs could be combined, if we used `cd` but this will change with the next rev anyways.